crates/bevy_wgpu/src/renderer/wgpu_render_resource_context.rs

```diff
@@ -252,9 +253,17 @@ impl RenderResourceContext for WgpuRenderResourceContext {
     fn create_shader_module_from_source(&self, shader_handle: &Handle<Shader>, shader: &Shader) {
         let mut shader_modules = self.resources.shader_modules.write();
         let spirv: Cow<[u32]> = shader.get_spirv(None).unwrap().into();
+        let source = wgpu::ShaderSource::SpirV(spirv);
+
+        let shader_module_descriptor = ShaderModuleDescriptor {
+            label: None,
+            source: source,
+            flags: Default::default()
```

Comment:
I think you'll probably want the shader validation flag to replicate the old behavior:

```suggestion
            flags: wgpu::ShaderFlags::VALIDATION
```